tests/testthat/test_bundle_step_umap.R

```diff
@@ -0,0 +1,41 @@
+test_that("bundling + unbundling step_umap", {
```

Comment:
Could this file be named in the same way as that in the `R/` folder? i.e. `test_bundle_embed.R`